mongodbatlas/resource_mongodbatlas_project_test.go

```diff
@@ -444,6 +470,17 @@ func testAccMongoDBAtlasProjectConfigWithProjectOwner(projectName, orgID, projec
 	`, projectName, orgID, projectOwnerID)
 }
 
+func testAccMongoDBAtlasGovProjectConfigWithProjectOwner(projectName, orgID, projectOwnerID string) string {
+	return fmt.Sprintf(`
+		resource "mongodbatlas_project" "test" {
+			name   			 = "%[1]s"
+			org_id 			 = "%[2]s"
+		    project_owner_id = "%[3]s"
+			is_mongodbgov_cloud = true
```

Comment:
since this is now at the provider level don't think this test applies anymore.  Need to update to region_usage_restrictions test.

Reply:
Changed test to use region_usage_restrictions parameter